docs/source/usage/workflows/boundaryConditions.rst

```diff
@@ -0,0 +1,44 @@
+.. _usage-workflows-boundaryConditions:
+
+Boundary Conditions
+-------------------
+
+.. sectionauthor:: Sergei Bastrakov
+
+There are two kinds of boundary conditions supported: periodic and absorbing.
+They are set in a :ref:`.cfg file <usage-tbg>` with option ``--periodic <x> <y> <z>``.
+Value 0 corresponds to absorbing boundaries along the axis (used by default), 1 corresponds to periodic.
+The same boundary condition kind is applied for all particles species and fields, on both sides.
+
+Particles
+"""""""""
+
+For particles, the boundaries always match the global simulation area border.
+The treatment of particles crossing a boundary is controlled by the ``boundaryCondition`` flag in :ref:`speciesDefinition.param <usage-params-core>`.
+Its default value provides the conventional treatment: particles crossing an absorbing boundary are deleted, particles crossing a periodic boundary are transferred to the other side of the global simulation area.
+The behavior for both boundary kinds can be customized per species by changing the flag to a user-defined type.
+
+Fields
+""""""
+
+Periodic boundary conditions for fields do not allow customization or variants.
+The rest of the secdion concerns absorbing boundaries.
+
+For the absorbing boundaries, there is a virtual field absorber layer inside the global simulation area near its boundaries.
+Field values in the layer are treated differently from the rest, by a combination of a field solver and a field absorber.
+It causes field dynamics inside the absorber layer to differ from that in a vacuum.
+Otherwise, the affected cells are a normal part of a simulation in terms of indexing, particle handling, and output.
+It is recommended to avoid, as possible, having particles in the field absorber layer.
```

Comment:
suggestion:
```suggestion
It is recommended to avoid, if possible, having particles in the field absorber layer.
```

Reply:
I'm not sure what's grammatically better. My idea of this plus the next sentense was that it's often unavoidable to have particles there. But ideally only have leaving particles and no "relevant" particles.

Reply:
Btw there are plans to change it so that particles are absorbed before the field absorber layer. But of course now it describes the current way.

Reply:
Okay - for me `as` "feels" uncommon/"unfamilar" - but let's keep it as it is.right now.